src/mixins/legal-api-mixin.ts

```diff
@@ -64,6 +64,21 @@ export default class LegalApiMixin extends Mixins(FilingTemplateMixin) {
       })
   }
 
+  /**
+   * Deletes a filing by its id.
+   * @returns a promise to delete the filing or return a failure.
+   */
+  async deleteFilingById (id: number): Promise<any> {
+    const url = `businesses/${this.getBusinessId}/filings/${id}`
+
+    return axios.delete(url)
+      .catch(error => {
+      // eslint-disable-next-line no-console
+        console.log('deleteFilingById() error - invalid response =', error)
+        throw new Error('Invalid API response')
```

Comment:
Should consider error handling in future ticket.

What should we do if deleting fails? Maybe just put up a dialog and let user proceed (just an OK button).

Reply:
This could probably use the same mechanism as "save-error-event".

Reply:
Copy that, will do!

Reply:
Updated in next commit 👍

Reply:
Actually, need to update this one more time.

Reply:
Need to create a new error event, as the other types don't fit here.

This is going to take some more work, will have to get it in the morning at this point, my day ends when the kids get home.